core/processor/src/main/java/io/quarkus/annotation/processor/generate_doc/SummaryTableDocFormatter.java

```diff
@@ -8,28 +8,35 @@
 
 final class SummaryTableDocFormatter implements DocFormatter {
     private static final String TABLE_CLOSING_TAG = "\n|===";
-    private static final String TABLE_ROW_FORMAT = "\n\na|%s `%s`\n\n[.description]\n--\n%s\n--|%s %s\n|%s\n";
-    private static final String TABLE_SECTION_ROW_FORMAT = "\n\nh|%s\nh|Type\nh|Default";
+    private static final String TABLE_ROW_FORMAT = "\n\na|%s [[%s]]`link:#%s[%s]`\n\n[.description]\n--\n%s\n--|%s %s\n|%s\n";
+    private static final String TABLE_SECTION_ROW_FORMAT = "\n\nh|[[%s]]link:#%s[%s]\nh|Type\nh|Default";
     private static final String TABLE_HEADER_FORMAT = "[.configuration-legend]%s\n[.configuration-reference, cols=\"80,.^10,.^10\"]\n|===";
     //    private static final String MORE_INFO_ABOUT_SECTION_FORMAT = "link:#%s[icon:plus-circle[], title=More information about %s]";
 
+    private String anchorPrefix = "";
+
     /**
      * Generate configuration keys in table format.
      * Generated table will contain a key column that points to the long descriptive format.
      *
      * @param configDocItems
      */
     @Override
-    public void format(Writer writer, List<ConfigDocItem> configDocItems) throws IOException {
+    public void format(Writer writer, boolean useAnchorPrefixes, List<ConfigDocItem> configDocItems) throws IOException {
```

Comment:
Is there any reason we can't generate anchors all the time?

Reply:
I think we do, but they don't need a prefix if we're generating a single page.